pkgs/servers/home-assistant/custom-components/easee/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  pyeasee,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "nordicopen";
+  domain = "easee";
+  version = "0.9.67";
+
+  src = fetchFromGitHub {
+    owner = "nordicopen";
+    repo = "easee_hass";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-psRr3cJ/sK/Z0dgB27GbW0qAHH2vJt+TdxqDB+Zhkc0=";
+  };
+
+  dependencies = [
+    pyeasee
+  ];
+
+  dontBuild = true;
```

Comment:
Why?

Reply:
The builder defaults to `true`, so this can indeed be dropped.

Reply:
I only tested this against 24.11, where this was required, thanks for pointing it out!